cyclonedx-lib/build.xml

```diff
@@ -69,6 +69,24 @@
                 </copy>
         </target>
 
+        <target name="build-sign-sbom" depends="dep-checks, clone-and-build-openkeystore, download-cyclonedx, download-jackson-core, download-jackson-dataformat-xml, download-jackson-databind, download-jackson-annotations, download-json-schema, download-commons-codec, download-commons-io, download-github-package-url, compile-sign-sbom, jar-sign-sbom">
```

Comment:
clone-and-build-openkeystore is not building it within the build folder, try this:
```
        <target name="clone-and-build-openkeystore" unless="openkeystore_available">
		            <mkdir dir="build"/>
		            <exec executable="git" dir="build">
			          <arg line="clone https://github.com/cyberphone/openkeystore" />
		            </exec>
		            <ant dir="build/openkeystore/library" target="build"/>
        </target>
```